robotoff/logos.py

```diff
@@ -258,12 +258,19 @@ def import_logo_insights(
 
 
 def generate_insights_from_annotated_logos(
-    logos: List[LogoAnnotation], server_domain: str
-):
+    logos: List[LogoAnnotation],
+    server_domain: str,
+) -> int:
     predictions = []
     for logo in logos:
         prediction = generate_prediction(
-            logo.annotation_type, logo.taxonomy_value, confidence=1.0, logo_id=logo.id
+            logo.annotation_type,
+            logo.taxonomy_value,
+            confidence=1.0,
+            logo_id=logo.id,
+            username=logo.username,
+            is_annotation=True,  # explicit is better than implicit
+            automatic_processing=True,  # this is a user annotation !
```

Comment:
```suggestion
            is_annotation=True,  # it's worth restating it
            automatic_processing=True,  # because this is a user annotation, which we trust.
```